src/assets/src/components/queue.tsx

```diff
@@ -207,8 +207,7 @@ function QueueAttendingJoined(props: QueueAttendingProps) {
     const meeting = props.queue.my_meeting!;
     const meetingBackend = getBackendByName(meeting.backend_type, props.backends);
     const isVideoMeeting = VideoBackendNames.includes(meetingBackend.name);
-    const numberInLine = meeting.line_place !== null ? meeting.line_place + 1 : null;
-    const inProgress = meeting.status === MeetingStatus.STARTED;
+    const inProgress = meeting.status === MeetingStatus.STARTED && meeting.line_place === null;
```

Comment:
I've been struggling with whether or not we actually need to check what's going on with `line_place` here if we've already checked the `status` (since whether `line_place` is `null` or not is dependent on `status`). The WebSocket updates change both at the same time. I guess it doesn't hurt anything to keep it like this, maybe it protects us a little against changes to backend logic? I'd be interested to hear others' opinions on this.

Reply:
My inclination is to try to keep most of the business logic in the backend so it doesn't have to be replicated in the frontend. This lets the frontend neatly reflect business logic changes without additional code changes in most circumstances. So I'd simplify this to just checking the `status`.
The variable naming `inProgress` also hints to me that `line_place` should have nothing to do with it -- or that the name should be adjusted. I could imagine changing `line_place` to be 0 or -1 or something if the meeting is started, in which case this code might stop working the way it should.

Reply:
Makes sense. I will simplify and test.

Reply:
Reworked a little bit after chatting with @jlost.